exercise/cache/exercise.py

```diff
@@ -1,4 +1,4 @@
-import time
+import time, lzma
```

Comment:
Could you split these to their own lines, see https://www.python.org/dev/peps/pep-0008/#imports